icontrol/session.py

```diff
@@ -292,8 +311,11 @@ def __init__(self, username, password, **kwargs):
             self.session.auth = (username, password)
 
         # Set state as indicated by ancestral code.
-        self.session.verify = False  # XXXmake TOFU
-        self.session.headers.update({'Content-Type': 'application/json'})
+        self.session.verify = False  # XXX make TOFU
+        self.session.headers.update({
```

Comment:
So now you set a user agent no matter what. Does that still work with the existing functional tests? Just making sure you ran all of the existing functional tests.